Refurbish

egabancho opened the pull request.

tmorrell commented on CONTRIBUTING.rst:
```diff
@@ -108,6 +113,6 @@ Before you submit a pull request, check that it meets these guidelines:
    your new functionality into a function with a docstring.
-3. The pull request should work for Python 2.7, 3.3, 3.4 and 3.5. Check
-   https://travis-ci.com/inveniosoftware/invenio-collections/pull_requests
+3. The pull request should work for Python 3.6, 3.7, 3.8 and 3.9. Check
```
Can we switch this to "The pull request should work for Python versions supported by Invenio."? We might want to do a global change on this....since many of them still mention Python 2.7

tmorrell replied:
We've got it [here](https://inveniordm.docs.cern.ch/install/requirements/), and CONTRIBUTING links below to the CI list, which is probably more likely to be updated anyway.

utnapischtim commented:
> I forgot about that! I have been using **only** `pyproject.toml` for a while now 😇

should we change this package to `pyproject.toml`? we do want to do it for the other packages anyway so why not start with this one?

egabancho commented:
> I think you should add in the MIT top level LICENSE file.

@tmorrell, What do you mean? I changed the content of `LICENSE` already, and replaced it with the MIT one.

tmorrell commented:
> > I think you should add in the MIT top level LICENSE file.
> 
> @tmorrell, What do you mean? I changed the content of `LICENSE` already, and replaced it with the MIT one.

Sorry, it looks good. I got confused by the GitHub diff. The old license was really long....
